about.html

```diff
@@ -0,0 +1,76 @@
+  <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+  <title>About us | CC Global Summit 2015</title>
+</head>
+<body>
+  <header class="ab-header">
+      <nav class="nav">
+        <div class="top-menu hidden">
+          <img src="./images/facebook.png" alt="facebook icon">
+          <img src="./images/twitter.png" alt="twitter icon">
+          <p class="lato">English</p>
+          <p class="lato">My Page</p>
+          <p class="lato">Logout</p>
+        </div>
+        <div class="menu-bar lato hidden" id="menu-bar">
+          <a href="./index.html" id="ccbutton"><img src="./images/about_logo.png" alt=""></a>
+          <a href="./index.html" class="menu-item">Home</a>
+          <a href="#" class="menu-item">About</a>          
+          <a href="#past" class="menu-item">Past Conventions</a>
+          <a href="https://www.comic-con.org/" class="menu-item">Sponsor</a>
+          <a href="https://comic-con.com.ar/" class="menu-item">News</a>
+        </div>
+        <button type="button" class="menuBtn" id="menuBtn"></button>
+        <button type="button" class="btnClose hidden" id="quitBtn"></button>
+      </nav>
+      <div class="ab-container">
+        <p class="lato headline">"Hello, COMICONERS!!!"</p>
+        <h1 class="abText cocogoose">COMIC CON 2022 - ARGENTINA</h1>
+        <p class="ab-textBox lato">The Global Summit brings together the community of experts, academics and activist who comprise the Creative Commons affiliate network in a different country every two.
+            This year, we hope to expand our invitation list including organizations and individuals.
+        </p>
+        <h2 class="contact lato">Please contact us per Email for any further question about CC Global Summit 2015!</h2>
+        <a class="lato mail" href="mailto:ccsumit2015@cckorea.org">ccsumit2015@cckorea.org</a>
+      </div>
+  </header>
+  <main class="ab-main lato">
+    <h3>Creative Commons Global Summit 2015 Logo</h3>
+    <hr>
+    <p class="ab-mainText lato">The logo of CC Global Summit 2015 was decided through the logo competition from 8.June to 7.July.</p>
+    <img src="./images/about_logo.png" alt="logo" >
+  </main>
+  <hr>
+  <article class="ab-article lato" id="past">
+    <h3>See the past CC Global Summits</h3>
+    <hr>
+    <p>Take a look at the last two CC Global Summits which look place in Buenos Aires and in Warsaw.</p>
+    <img src="./images/about_2011.png" alt="past cc global submit 2011">
+    <img src="./images/about_2013.png" alt="past cc global submit 2013">
+  </article>
+  <article class="ab-artPartner lato">
+    <h3>Partner</h3>
+    <hr>
+    <div class="sponsors">
+      <img src="./images/marvel.png" alt="marvel">
+      <img src="./images/googlekorea.png" alt="google">
+      <img src="./images/mozilla.png" alt="mozilla">
+      <img src="./images/advengers.png" alt="advengers">
+      <img src="./images/dc.png" alt="dc">
+    </div>
```

Comment:
- [ ] Here, your images are being shown on the screen but they are too big, so in order to stick to the original design as much as possible please reduce the size.